manifests/params.pp

```diff
@@ -64,4 +64,6 @@
 
   $qpid_url = "amqp:ssl:${::fqdn}:5671"
   $candlepin_event_queue = 'katello_event_queue'
+
+  $is_parent = true
```

Comment:
params.pp is only if the parameter is actually a class parameter (in this case on katello  itself).